tracing/init.go

```diff
@@ -1,31 +1,62 @@
 package tracing
 
 import (
-	"io"
+	"context"
+	"fmt"
+	"log"
+	"time"
 
-	opentracing "github.com/opentracing/opentracing-go"
-	jaeger "github.com/uber/jaeger-client-go"
-	jaegerCfg "github.com/uber/jaeger-client-go/config"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/sdk/resource"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
+	"go.opentelemetry.io/otel/trace"
 )
 
 // JaegerCollectorURL defines the address of Jaeger collector to submit spans.
 var JaegerCollectorURL = "http://localhost:14268/api/traces"
```

Comment:
Another option is to start using OTLP as exporter instead of Jaeger format, because Jaeger backend can already accept it. Or make it an option to switch between `jaeger` and `otlp`. But not in this PR.